app/models/hosting_environment.rb

```diff
@@ -11,6 +11,10 @@ def self.aytq_domain
     host
   end
 
+  def self.check_records_domain
+    ENV.fetch("CHECK_RECORDS_DOMAIN", "localhost")
```

Comment:
This is a temporary fallback until the prod-like environments are correctly setting CHECK_RECORDS_DOMAIN when being deployed.

Reply:
I think it's a good idea, definitely worth discussing. I'd like to not use Azure for this at all, but I think there'd be some resistance from the wider team to deviating from that model. Especially as production env var settings are locked down by Azure's access control.